### PR TITLE
[Feature] Add default text when search is empty

### PR DESCRIPTION
**Pull request by aashay28**
## Feature
- Add default text when search is empty

## Screenshots

Screenshot 2023-04-26 at 12 08 19 PM

## Note to reviewers

feature has been updated and tested its working perfect.
please verify the feature and let me know if there is any other modification required.
thank you

### CODE REVIEW

File: components/SideNavbar/SideNavbarCategoryList.tsx [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+          return (
+            <SideNavbarCategory
+              key={index}
+              item={item}
+              openByDefault={openByDefault}
+            />
+          )
+        })
+      ) : (
+        <div className="text-slate-500 py-2">No Links Found</div>
```
Comment (rupali-codes) [inline components/SideNavbar/SideNavbarCategoryList.tsx:24]:
```suggestion
        <div className="dark:text-gray-200 text-gray-500 text-lg text-center py-2">No Links Found</div>
```

File: components/SideNavbar/SideNavbarCategoryList.tsx [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+import { FC } from 'react'
+import type { ISidebar } from '../../types'
+import { SideNavbarCategory } from './SideNavbarCategory'
 
+export const SideNavbarCategoryList: FC<{
+  items: ISidebar[]
+  openByDefault: string
+}> = (props) => {
+  const { items, openByDefault } = props
+  console.log('items', items)
```
Comment (imbeshat) [inline components/SideNavbar/SideNavbarCategoryList.tsx:10]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Maybe you forgot to remove ```console.log``` after testing

**Comment by aashay28** [comment]
> Maybe you forgot to remove `console.log` after testing

yes my bad !!
thanks

**Review by rupali-codes — APPROVED** [review]
lgtm, thanks

**Pull request merged**